modules/multi2vec-google/clients/google.go

```diff
@@ -170,7 +170,7 @@ func (v *palm) getPayload(text, img, vid string, config ent.VectorizationConfig)
 	return req
 }
 
-func (v *palm) checkResponse(statusCode int, palmApiError *palmApiError) error {
+func (v *google) checkResponse(statusCode int, palmApiError *palmApiError) error {
```

Comment:
just minor renaming `googleApiError *googleApiError`

Reply:
yes, will do that!